controllers/vcdmachine_controller.go

```diff
@@ -441,7 +449,10 @@ func (r *VCDMachineReconciler) reconcileNormal(ctx context.Context, cluster *clu
 		cloudInitInput.HTTPProxy = vcdCluster.Spec.ProxyConfigSpec.HTTPProxy
 		cloudInitInput.HTTPSProxy = vcdCluster.Spec.ProxyConfigSpec.HTTPSProxy
 		cloudInitInput.NoProxy = vcdCluster.Spec.ProxyConfigSpec.NoProxy
-		cloudInitInput.MachineName = machine.Name
+		cloudInitInput.MachineName = vmName
+		if err != nil {
```

Comment:
It is not sure what error is being checked here. Can you please make sure this is not added by mistake?

Reply:
@erkanerol could you please handle this? we should be able to merge immediately after it

Reply:
It is leftover after a refactoring. Just deleted the check. Thanks for catching this.